ansible/roles/analytics-druid/defaults/main.yml

```diff
@@ -200,23 +200,23 @@ default_druid_configs:
    druid_postgres_user: "{{ druid_postgres_user | default('druid@' + postgres.db_url) }}"
    #Druid Azure Details
    druid_postgres_pass: "{{ dp_vault_druid_postgress_pass }}"
-   azure_account_name: "{{ sunbird_druid_storage_account_name }}"
```

Comment:
`sunbird_druid_storage_account_name` is this an internal variable or a common variable?